docs/reference/run.md

```diff
@@ -1308,6 +1308,10 @@ or two examples of how to pass more parameters to that ENTRYPOINT:
 > **Note**: Passing `--entrypoint` will clear out any default command set on the
 > image (i.e. any `CMD` instruction in the Dockerfile used to build it).
 
+> **Note**: Passing `--entrypoint=""` (empty string) will reset the entry
+> point to system default (i.e., the entry point used by docker when there
+> is no `ENTRYPOINT` instruction in the Dockerfile).
```

Comment:
I don't think this should be a note (as it's not an "unexpected" behavior) perhaps similar to the flag description;

```Markdown
You can reset a containers entrypoint by passing an empty string, for example:

     $ docker run -it --entrypoint="" mysql bash

```

(and move that above the `**Note**`)